integtest/src/main/java/org/corfudb/universe/cluster/docker/FakeDns.java

```diff
@@ -0,0 +1,180 @@
+                        nameServiceInterface.getClassLoader(),
+                        new Class<?>[]{nameServiceInterface},
+                        new NameServiceListener(fallbackNameService)
+                );
+                field.setAccessible(true);
+                field.set(InetAddress.class, proxy);
+            } catch (final ClassNotFoundException | NoSuchFieldException e) {
+                // Override the NameService in Java 8 or earlier.
+                final Class<?> nameServiceInterface = Class.forName("sun.net.spi.nameservice.NameService");
+                Field field = InetAddress.class.getDeclaredField("nameServices");
+                // Get the default NameService to fallback to.
+                Method method = InetAddress.class.getDeclaredMethod("createNSProvider", String.class);
+                method.setAccessible(true);
+                Object fallbackNameService = method.invoke(null, "default");
+                // Create a proxy instance to set on the InetAddress field which will handle
+                // all NameService calls.
+                Object proxy = Proxy.newProxyInstance(
+                        nameServiceInterface.getClassLoader(),
+                        new Class<?>[]{nameServiceInterface},
+                        new NameServiceListener(fallbackNameService)
+                );
+                field.setAccessible(true);
+                // Java 8 or earlier takes a list of NameServices
+                field.set(InetAddress.class, Arrays.asList(proxy));
+            }
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+        installed = true;
+
+        return this;
+    }
+
+    /**
+     * The NameService in all versions of Java has the same interface, so we
+     * can use the same InvocationHandler as our proxy instance for both
+     * java.net.InetAddress$NameService and sun.net.spi.nameservice.NameService.
+     */
+    private class NameServiceListener implements InvocationHandler {
+
+        private final Object fallbackNameService;
+
+        // Creates a NameServiceListener with a NameService implementation to
+        // fallback to. The parameter is untyped so we can handle the NameService
+        // type in all versions of Java with reflection.
+        NameServiceListener(Object fallbackNameService) {
+            this.fallbackNameService = fallbackNameService;
+        }
+
+        private InetAddress[] lookupAllHostAddr(String host) throws UnknownHostException {
+            InetAddress inetAddress;
+            synchronized (FakeDns.this) {
+                inetAddress = forwardResolutions.get(host);
+            }
+            if (inetAddress != null) {
+                return new InetAddress[]{inetAddress};
+            }
+
+            try {
+                Method method = fallbackNameService.getClass().getDeclaredMethod("lookupAllHostAddr", String.class);
+                method.setAccessible(true);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make sure that this accessibility update is safe here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3011)

Reply:
Don't see ways how to fix this